pulp_docker/app/tasks/sync_stages.py

```diff
@@ -215,25 +247,11 @@ def __init__(self, remote):
             extra_data={'headers': V2_ACCEPT_HEADERS}
         )
         man_dc = DeclarativeContent(content=manifest, d_artifacts=[da])
-        for layer in manifest_data.get('layers'):
-            if not self._include_layer(layer):
-                continue
-            blob_dc = await self.create_pending_blob(layer)
-            blob_dc.extra_data['relation'] = man_dc
-            await self.put(blob_dc)
-        layer = manifest_data.get('config')
-        if layer and self._include_layer(layer):
-            config_blob_dc = await self.create_pending_blob(layer)
-            config_blob_dc.extra_data['config_relation'] = man_dc
-            await self.put(config_blob_dc)
-        man_dc.extra_data['relation'] = tag_dc
-        tag_dc.extra_data['processed'] = True
-        man_dc.extra_data['processed'] = True
-        await self.put(man_dc)
-
-    async def create_pending_manifest(self, list_dc, manifest_data):
+        return man_dc
+
+    def create_manifest(self, list_dc, manifest_data):
         """
-        Create a pending manifest from manifest data in a ManifestList.
+        Create a Manifest from manifest data in a ManifestList.
```

Comment:
More generally, since this data is coming in, it doesn't matter where its from to this code.

Reply:
i wanted to stress that this manifest is not tagged and when creating it, we rely on some information provided from ML